v5_client_web_socket_service.go

```diff
@@ -38,7 +44,13 @@ func (s *V5WebsocketService) Public(category CategoryV5) (V5WebsocketPublicServi
 // Private :
 func (s *V5WebsocketService) Private() (V5WebsocketPrivateServiceI, error) {
 	url := s.client.baseURL + V5WebsocketPrivatePath
-	c, _, err := websocket.DefaultDialer.Dial(url, nil)
+	var c *websocket.Conn
+	var err error
+	if s.client.dialer != nil {
+		c, _, err = s.client.dialer.Dial(url, nil)
+	} else {
+		c, _, err = websocket.DefaultDialer.Dial(url, nil)
+	}
```

Comment:
It looks good functionally 👍

As an implementation, there may be room for refactoring.
Because the same writing style is used in multiple places